hlc_wp_mg3/config.cpp

```diff
@@ -642,6 +642,15 @@ class CfgMagazines{
     };
 };
 
+class CfgMagazineWells {
+    class CBA_792x57_LINKS {
+        NIA_belts[] = { __792MG42_MAGS };
+    };
+    class CBA_762x51_LINKS {
+        NIA_MG3_belts[] = { __762MG3_MAGS };

```

Comment:
These are using DM6 links, not M13. Should be a separate magwell class, MG3 weapons having both or only their own one. See https://github.com/toadie2k/NIArms/pull/79#issuecomment-324221046